hs_core/management/commands/reorder_authors.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+
+"""
+Fix duplicate author "order" values
+
+Related to https://github.com/hydroshare/hydroshare/issues/4695
+"""
+
+from django.core.management.base import BaseCommand
+from hs_core.models import BaseResource
+
+
+class Command(BaseCommand):
+    help = "Fix duplicate author 'order' values"
+
+    def handle(self, *args, **options):
+        resources = BaseResource.objects.filter(raccess__published=False).only('object_id', 'short_id')
+        for res in resources:
+            # creators = Creator.objects.filter(object_id=res.object_id)
+            creators = res.metadata.creators.all()
```

Comment:
There may be corrupted resources that may not have the `metadata` set for the resource. I know at least there is one. So basically need to check `res.metadata is not None` before accessing the `creators`.  The other way is to use line 19 instead.